### PR TITLE
[HOLD] - [Refactor] Fix deprecated fields: PartnerShow -> Show

### PR DESCRIPTION
**Pull request by damassi**
Blocked until https://github.com/artsy/metaphysics/pull/1776 is in. I was able to complete this work by synching a locally-running shema from 1776, so once that's merged I can do a proper sync from upstream. 

Fixes some deprecated field references, namely `PartnerShow` -> `Show`. 

Screen Shot 2019-06-14 at 1 20 24 PM

Since I'm not intimately familiar with the PartnerShow / Show schema, not sure if there's anything else to do, but things seemed to type-check just fine.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 860118473ddacf049c559734a395129b54dcb62c

https://deploy-preview-2552--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const ArtworkSidebarClassificationFragmentContainer = createFragmentConta
     artwork: graphql`
       fragment ArtworkSidebarClassification_artwork on Artwork {
         attribution_class {
-          short_description
+          shortDescription
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx]: Nice, was this a bug?

Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx]: Nah, just a separate deprecation. With our new Apollo tooling this stuff jumps out in VSCode so was able to easily fix.

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/FairArtworkGrid.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export const FairArtworkGridFragmentContainer = createFragmentContainer(
         fair: show(at_a_fair: true) {
           href
 
-          artworksConnection(first: 8, exclude: $excludeArtworkIDs) {
+          artworks_connection(first: 8, exclude: $excludeArtworkIDs) {
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/FairArtworkGrid.tsx]: Subtle, `Show` has an underscore-ized field.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx (hunk 1 above)
Comment (zephraph) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx]: We had a pattern of `_` names which we've deprecated in favor of camel case.

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/FairArtworkGrid.tsx (hunk 2 above)
Comment (alloy) [reply src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/FairArtworkGrid.tsx]: 😑

**Comment by damassi** [comment]
Going to close this for now

**Pull request closed without merging**